chat-commands.js

```diff
@@ -2274,9 +2274,11 @@ exports.commands = {
 			target = targets[1].trim();
 			roomId = toId(targets[0]) || room.id;
 		}
+		let targetRoom = Rooms.get(roomId);
+		let addModlogLinks = Config.modloglink && (!hideIps || (targetRoom && !targetRoom.isPrivate));
```

Comment:
oops, I should've edited the commit message when I removed Config.roomauthModlogLink. Well, it's too late now.